test/translation-wrapper-mocha.js

```diff
@@ -0,0 +1,253 @@
+                                  translationWrapper({ inPorts:{ input: { required: true } },
+                                                 updater: function(input) {
+                                                     return input;
+                                                 }
+                                  })
+                             }
+                }).then(function(network) {
+                    return new Promise(function(done) {
+
+                        var nodeA = network.processes.nodeA.component;
+                        var nodeS = network.processes.nodeS.component;
+
+                        var countSends = function() {
+                            count++;
+                            if (count === 3) {
+                                done();
+                            }
+                        };
+                        commonTest.onOutPortData(nodeS, 'output', countSends);
+
+			network.graph.addEdge('nodeA', 'output', 'nodeS', 'input');
+
+                        network.graph.addInitial('{"1":"one","2":"two","3":"three"}', 'nodeA', 'input');
+                        setTimeout(done, 1000);
+
+                    }).then(function(done) {
+                        count.should.equal(3);
+                    });
+                });
+            });
+
+            it("should call updater with extra parameters", function() {
+                this.timeout(4250);
+                var testString = 'This little piggy stayed home';
+                return commonTest.createNetwork({
+                    nodeA: 'rdf-components/parse-json',
+                    nodeS: translationWrapper({
+                        updater: function(input, arg) {
+                            return arg;
+                        }
+                    })
+                }).then(function(network) {
+                    return new Promise(function(done) {
+
+                        var nodeA = network.processes.nodeA.component;
+                        var nodeS = network.processes.nodeS.component;
+
+                        commonTest.onOutPortData(nodeS, 'output', done);
+
+                        network.graph.addEdge('nodeA', 'output', 'nodeS', 'input');
+
+                        network.graph.addInitial(testString, 'nodeS', 'arg');
+                        network.graph.addInitial('{"1":"one"}', 'nodeA', 'input');
+
+                    }).then(function(done) {
+                        done.vnid.should.equal('1');
+                        done.data.should.equal(testString);
+                        expect(done.error).to.be.undefined;
+                        expect(done.stale).to.be.undefined;
+                        done.groupLm.match(/^LM(\d+)\.(\d+)$/).should.have.length(3);
+                        done.lm.match(/^LM(\d+)\.(\d+)$/).should.have.length(3);
+                        done.groupLm.should.not.equal(done.lm);
+                    });
+                });
+            });
+
+       });
+   });
+});
```

Comment:
These tests are nearly identical to split-wrapper-mocha.js.  Could you do something like I did for the common-wrapper tests, to share this code?  Maybe a common-split-wrapper?